### PR TITLE
MQE-1959: Static-checks command can be configured

### PR DESCRIPTION
**Pull request by jilu1**
### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/commands/mftf.md [hunk 1]
```diff
@@ -463,18 +463,39 @@ The example parameters are taken from the `etc/config/.env.example` file.
...
+To run all existing static check scripts 
 
 ```bash
 vendor/bin/mftf static-checks
 ```
 
+To run specific static check scripts
+
+```bash
+vendor/bin/mftf static-checks testDependencies
```
Comment (tomreece) [inline docs/commands/mftf.md]: When I run this command I get:

```
bin/mftf static-checks testDependencies

Invalid static check script(s): testDependencies. Please fix input arguments and rerun.
```

Comment (jilu1) [reply docs/commands/mftf.md]: Good catch! Fixed.

Comment (tomreece) [reply docs/commands/mftf.md]: I can pass a single value now.

But when I try to pass two values it says the second one doesn't exist:

```
$ bin/mftf static-checks reece testDependencies

Invalid static check script(s): testDependencies. Please fix input arguments and rerun.
```

Try to pass two values and it does not work.

Comment (jilu1) [reply docs/commands/mftf.md]: Fixed.

**Pull request merged**